client/js/drawModule.js

```diff
@@ -0,0 +1,1031 @@
+        time = properties[i].timestamp;
+      }
+    }
+
+    var nDiners = document.getElementById('partySize').valueAsNumber;
+
+    var available = (capacity - occupancyLevel - nDiners) >= 0;
+
+
+    document.getElementById('loadingTime').style.visibility = 'hidden';
+
+    //if available allow to submit
+    document.getElementById('submitReservation').disabled =
+      ! available;
+
+      if (! available) {
+        var errMsg =
+          (capacity - occupancyLevel) +
+          ' seats available for the specified date and time';
+        utils.showMessage(errMsg, 'alert-warning');
+      }
+
+  }
+
+
+  function _errorProcesingAvailability() {
+    alert('An error ocurred checking reservation availability');
+    document.getElementById('loadingTime').style.visibility = 'hidden';
+  }
+
+  //create a tables for displaying the reviews
+  function createReviewsTable(reviewsResponse) {
+    reviewsResponse = JSON.parse(reviewsResponse);
+
+    //clean previous table content
+    var myNode = document.getElementById('reviewsTableBody');
+    myNode.innerHTML = '';
+
+    if (reviewsResponse.length < 1) {
+      var error = document.createElement('TR');
+      error.textContent = 'No reviews are available.';
+      document.getElementById('reviewsTableBody').appendChild(error);
+      return;
+    }
+
+    //add entries
+    reviewsResponse.forEach(createReviewsTableEntry);
+  }
+
+  function createReviewsTableEntry(review) {
+    var row = document.createElement('TR');
+    var name = document.createElement('TD');
+    name.textContent = review.itemReviewed.name;
+    name.className = 'col-xs-4';
+    row.appendChild(name);
+
+    var rating = document.createElement('TD');
+    rating.textContent = review.reviewRating.ratingValue;
+    rating.className = 'col-xs-2';
+    row.appendChild(rating);
+
+    var view = document.createElement('TD');
+    view.className = 'col-xs-2';
+
+    var viewLink = document.createElement('A');
+    viewLink.textContent = 'View review';
+    viewLink.onclick = createViewReviewLink(review.name);
+
+    view.appendChild(viewLink);
+    row.appendChild(view);
+
+    var edit = document.createElement('TD');
+    edit.className = 'col-xs-2';
+
+    var editLink = document.createElement('A');
+    editLink.textContent = 'Edit review';
+    editLink.onclick = createEditReviewLink(review.name);
+
+    edit.appendChild(editLink);
+    row.appendChild(edit);
+
+    var del = document.createElement('TD');
+    del.className = 'col-xs-2';
+
+    var delLink = document.createElement('A');
+    delLink.textContent = 'Delete review';
+    delLink.onclick = createDelReviewLink(review.name);
+
+    del.appendChild(delLink);
+    row.appendChild(del);
+
+    document.getElementById('reviewsTableBody').appendChild(row);
+  }
+
+  //create a table for displaying the organization reviews
+  //different from createReviewsTable because this does not
+  //allow to modify or delete a review.
```

Comment:
"//create a table for displaying the organization reviews
//different from createReviewsTable because this does not
//allow to modify or delete a review." -> 
"//create a table for displaying the organization reviews. 
//The new table is different from createReviewsTable because this does not
//allow to modify or delete a review."